web/styles/right_sidebar.css

```diff
@@ -86,19 +112,60 @@ $user_status_emoji_width: 24px;
     }
 
     .empty-list-message {
-        font-size: 1.2em;
+        font-style: italic;
+        color: hsl(0deg 0% 67%);
```

Comment:
It might be nice to put this in a CSS variable, too, along with the other colors.

Reply:
Yeah sure! I was grabbing it from the old empty message styling